pkg/features/kube_features.go

```diff
@@ -406,7 +406,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	// inherited features from generic apiserver, relisted here to get a conflict if it is changed
 	// unintentionally on either side:
 	genericfeatures.StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
-	genericfeatures.AdvancedAuditing:        {Default: true, PreRelease: utilfeature.Beta},
+	genericfeatures.AdvancedAuditing:        {Default: true, PreRelease: utilfeature.GA},
```

Comment:
@loburm I think now based on https://github.com/kubernetes/website/pull/10294 we should drop that feature gate.

Reply:
Correct me if I'm wrong, but from my understanding now it's in GA and feature gate should be just deprecated (it's already no-op), and then after 6 months (2 releases) we can remove it.

Reply:
Right, was it deprecated in that case? At least mention that in the release notes and update docs. Although I think usually we should print the deprecation warning, I think it's worth adding to 1.12.1.

Reply:
Sorry I have missed this. Shall we deprecate it in 1.12.2 (not sure that policy allows us to do it in patch releases)? I would prefer to leave it for 1.13. WDYT?